### PR TITLE
Sail update

### PR DESCRIPTION
**Pull request by JRPearson500**

### CODE REVIEW

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+---
+remote_theme: nhsx/nhsx-io-theme
+title: Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank
+description: NHSX PhD Internship - Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank
+permalink: /Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: Where possible I shortened the slugs, so have done this here as well.

`/Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/`
-> `/transforming-healthcare-data-graph-based-sail/`

File: index.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ Below is a list of projects which are continually being added to. We welcome pro
 - [NHS GreenSpace](https://nhsx.github.io/nhsx-internship-projects/nhs-greenspace/)
 - [NHS Language Corpus](https://nhsx.github.io/nhsx-internship-projects/nhs-language-corpus/)
 - [NHS Semantic Search](https://nhsx.github.io/nhsx-internship-projects/nhs-semantic-search/)
-- [Predicting outcomes using SAIL Databank](https://nhsx.github.io/nhsx-internship-projects/sail-databank-outcomes-prediciton/)
+- [Transforming Healthcare Data with Graph-based Techniques Using SAIL DataBank](https://nhsx.github.io/nhsx-internship-projects/Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/)
```
Comment (danjscho) [inline index.md]: Where possible I shortened the slugs, so have done this here as well.

`/Transforming-Healthcare-Data-with-Graph-Based-Techniques-Using-SAIL-DataBank/`
-> `/transforming-healthcare-data-graph-based-sail/`

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+# Graph 
+
+**Keywords:** Graph Structures, Data Representation, Hypergraphs
+
+**Need:** The collections of linked health datasets into a single research environment creates a number of large opportunities for insight and analysis.  Whilst anonymised extracts from these environments have great value, there is an even larger opportunity to develop methods of deploying modelling to the data remotely.  [The SAIL Databank - The Secure Anonymised Information Linkage Databank](https://saildatabank.com/) provides anonymised person-based data for research powered by the Secure e-Research Platform.  
+
+Recent work using graph models to build simpler knowledge discovery systems opens up potential for increased prediction accuracies, reduced pre-processing burden and the application of higher complexities models to our data.  These models have been shown to effectively handle messy data and to learn representation of key factors from the data directly (rather than choosing a set of predictor variables).
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: `higher complexities models` -> `models of higher complexity`

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+**Current Knowledge/Examples & Possible Techniques/Approaches:** The recent paper entitled [Ranking Sets of Morbidities using Hypergraph Centrality](https://www.sciencedirect.com/science/article/pii/S1532046421002458) and accompanying [github repo] (https://github.com/jim-rafferty/multimorbidity_hypergraphs) demonstrates applying a hypergraph analysis to a comorbidity question.  Where possible, this project would seek to develop upon this work looking at including further complexity through adding demographics, direction into the graphs, or time evolution.  A likely first step to this would be a comparison of hypergraphs application versus state-models and more traditional clustering.
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: `and more traditional clustering.` -> `and more traditional clustering approaches.`

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+Information on the data contained within the SAIL Databank can be found on their site - [Sail-Datasets](https://saildatabank.com/saildata/sail-datasets/).
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: `[Sail-Datasets]` -> `[SAIL Datasets]`

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+A list of previous projects can be found here: [Sail-Previous-Projects](https://saildatabank.com/wp-content/uploads/Current-Projects-Jul-2021.pdf) and potential use-cases here: [Ssil-Use Cases](https://saildatabank.com/saildata/uses-for-sail-data/).
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: `[Sail-Previous-Projects]` -> `[SAIL Previous Projects]`
`[Ssil-Use Cases]` -> `[SAIL Use Cases]`

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+Information on the data contained within the SAIL Databank can be found on their site - [Sail-Datasets](https://saildatabank.com/saildata/sail-datasets/).
+A list of previous projects can be found here: [Sail-Previous-Projects](https://saildatabank.com/wp-content/uploads/Current-Projects-Jul-2021.pdf) and potential use-cases here: [Ssil-Use Cases](https://saildatabank.com/saildata/uses-for-sail-data/).  
+
+**Related Previous Internship Projects:** n/a
+
+**Enables Future Work:** This work would fit within a potential longer term three year research collaboration between SAIL and NHSX.  This longer term project would seek to recreate some graph neural network embeddings similar to work discussed in [Variationally Regularized Graph-based Representation Learning for Electronic Health Records](https://arxiv.org/abs/1912.03761), [Deep Patient: An Unsupervised Representation to Predict the Future of Patients from the Electronic Health Records](https://pubmed.ncbi.nlm.nih.gov/27185194/), and [Scalable and accurate deep learning with electronic health records](https://www.nature.com/articles/s41746-018-0029-1)
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: `would seek to recreate some graph neural network embeddings` -> `would seek to explore graph neural network (GNN) approaches`

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+**Outcome/Learning Objectives:** A comparison of hypergraphs applied to SAIL data versus state-models and clustering techniques to demonstrate the value of the graph application.  An extension of this would be to add a single demographic variable to each technique and show the potential of these additional data.
+
+**Datasets:** SAIL Databank Datasets (Core-set)
+
+**Desired skill set:** When applying please highlight any experience around graph databases and representations, Graph-NN, Multi-State models or Clustering Algorithms, coding experience (including any coding in the open), and any other data science experience you feel relevant.
```
Comment (danjscho) [inline projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: `Graph-NN` -> `GNNs`

**Review by danjscho — CHANGES_REQUESTED** [review]

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 1 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

File: index.md (hunk 2 above)
Comment (JRPearson500) [reply index.md]: changed

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 3 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 4 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 5 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 6 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 7 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

File: projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md (hunk 8 above)
Comment (JRPearson500) [reply projects/transforming-healthcare-data-with-graph-based-techniques-using-sail-databank.md]: changed

**Review by danjscho — APPROVED** [review]

**Pull request merged**